contracts/Bridge/token/TokenController.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+import { IAccessControlManagerV8 } from "@venusprotocol/governance-contracts/contracts/Governance/IAccessControlManagerV8.sol";
+import { Pausable } from "@openzeppelin/contracts/security/Pausable.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { ensureNonzeroAddress } from "../../lib/validators.sol";
+
+contract TokenController is Ownable, Pausable {
+    address public accessControlManager;
+    mapping(address => bool) internal _blacklist;
+    mapping(address => uint256) internal minterToCap;
+    mapping(address => uint256) internal minterToMintedAmount;
```

Comment:
If we're going to extend this contract, we could add a gap variable

Reply:
We will inherit this contract and it won't be upgradeable so I think we can avoid adding gap here